kore/test/Test/Kore/Step/Simplification/Not.hs

```diff
@@ -35,13 +43,17 @@ import Kore.Variables.UnifiedVariable
 
 import qualified Test.Kore.Step.MockSymbols as Mock
 import Test.Kore.Step.Simplification
+import Test.Tasty.HUnit.Ext
```

Comment:
`Test.Tasty.HUnit.Ext` re-exports `Test.Tasty.HUnit`, with the intent that we would no longer directly import the latter.